addons/test/misra/misra-ctu-test.h

```diff
@@ -18,4 +18,6 @@ void misra_8_7_external(void);
 // cppcheck-suppress misra-c2012-2.5
 #define MISRA_2_5_VIOLATION 0
 
+// #12362
+extern void misra_8_7_compliant( void );
```

Comment:
hmm I have the feeling the `misra_8_7` function should be declared here also with a similar declaration.. I wonder if it was by mistake or by intention to not declare it. What happens if it's declared?